.github/workflows/android_cd.yml

```
@@ -68,6 +68,10 @@ jobs:
             - name: Debug Local Properties
               run: cat local.properties
 
+            - name: Extract Version Name
+              run: echo "##[set-output name=version;]v$(echo '${{ github.event.pull_request.title }}' | grep -oP 'release v\K[0-9]+\.[0-9]+\.[0-9]+')"
```

actionlint (1.7.4)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)